### ISSUE TITLE
Censorship - A perfectly valid draw is being replaced! 

### ISSUE THREAD

**Issue by ghunti** [airbloc/publicsale-lottery#1]
Hi.
You guys did a perfectly valid lottery draw. You explained why technically the lottery was as good (or even better) as picking the addresses one at a time.
All the people that was left out is not happy with the draw, so you redo the lottery?
It's easier to silence a minority (1500 first lucky winners), then an angry mob (the unlucky ones), that's for sure, but the best decision will always be the correct one, no matter who you anger!

You are building on a tech that's supposed to fight censorship, yet, your first brick is full of blood!

**Comment by rollbackpt** [comment]
Agree! Changing things due to peer pressure is never a good sign...
